src/main/java/frc/robot/commands/shooter/flywheel/ShooterFlywheelShuttleCommand.java

```diff
@@ -70,16 +72,15 @@ public void initialize() {
 
     /** Is robot in center line. */
     public boolean isFinished() {
-
-        return (poseSupplier.getPose2d().getX() < 326.365 + 70 || poseSupplier.getPose2d().getX() > 326.36 - 70);
+        return (poseSupplier.getPose2d().getX() < Units.inchesToMeters(326.365 + 70) || poseSupplier.getPose2d().getX() > Units.inchesToMeters(326.36 - 70));
     }
 
     /** What the command has to execute. */
     public void execute() {
 
         swerveSubsystem.setRobotRelativeDrivePowers(0, 0, -MathUtil.clamp(
```

Comment:
Your command will work, but it prevents the driver from translating while this happens. This will require a rework so swerve is still drivable.